llvm/test/CodeGen/AArch64/GlobalISel/select-bitfield-insert.ll

```diff
@@ -155,3 +155,15 @@ bb:
   store i64 %tmp4, ptr %p
   ret i64 %out
 }
+
+define <2 x i16> @pr63826(<2 x i16> %vec) {
```

Comment:
Any reason why this is living in the bitfield-insert IR test?

Reply:
Oh, I didn't find the most appropriate file to add this use case, but this file name has a **insert** postfix, so I'm pleasure to move this case to another more appropriate  file. do you have some suggestion?
Anyhow, I just want a case show the **gloabl ISel** and **DAG Isel** have same output assemble.

Reply:
Yes, the test itself is a good idea. I think it can go into it's own test file if there's no existing IR test file you can find. Using PR number is also ok, like `AArch64/GlobalISel/insert-vector-elt-pr63826.ll`